src/lib.rs

```diff
@@ -773,6 +773,32 @@ pub enum SwizzleComponent {
     W = 3,
 }
 
+impl SwizzleComponent {
```

Comment:
can this impl be moved to `proc/mod.rs` ? I'd like to keep the main `lib.rs` with only the definitions